operators/CrunchyData_postgres-operator/results.csv

```diff
@@ -0,0 +1,259 @@
+Trial number,Testcase,Alarm,Crash,Health,Operator log,Consistency,Differential,Custom,Is true alarm,CR change,Comment
```

Comment:
Added result analysis report as extra columns in this csv since I find this easier to track and read.